identifiers/country-au/federal_electorates.csv

```diff
@@ -1,151 +1,162 @@
+ocd-division/country:au/state:qld/federal_electorate:leichhardt,Queensland - Federal House of Representatives Leichhardt electorate,,
+ocd-division/country:au/state:qld/federal_electorate:lilley,Queensland - Federal House of Representatives Lilley electorate,,
+ocd-division/country:au/state:qld/federal_electorate:longman,Queensland - Federal House of Representatives Longman electorate,,
+ocd-division/country:au/state:qld/federal_electorate:maranoa,Queensland - Federal House of Representatives Maranoa electorate,,
+ocd-division/country:au/state:qld/federal_electorate:mcpherson,Queensland - Federal House of Representatives McPherson electorate,,
+ocd-division/country:au/state:qld/federal_electorate:moncrieff,Queensland - Federal House of Representatives Moncrieff electorate,,
+ocd-division/country:au/state:qld/federal_electorate:moreton,Queensland - Federal House of Representatives Moreton electorate,,
+ocd-division/country:au/state:qld/federal_electorate:oxley,Queensland - Federal House of Representatives Oxley electorate,,
+ocd-division/country:au/state:qld/federal_electorate:petrie,Queensland - Federal House of Representatives Petrie electorate,,
+ocd-division/country:au/state:qld/federal_electorate:rankin,Queensland - Federal House of Representatives Rankin electorate,,
+ocd-division/country:au/state:qld/federal_electorate:ryan,Queensland - Federal House of Representatives Ryan electorate,,
+ocd-division/country:au/state:qld/federal_electorate:wide_bay,Queensland - Federal House of Representatives Wide Bay electorate,,
+ocd-division/country:au/state:qld/federal_electorate:wright,Queensland - Federal House of Representatives Wright electorate,2009,
+ocd-division/country:au/state:sa/federal_electorate:adelaide,South Australia - Federal House of Representatives Adelaide electorate,,
+ocd-division/country:au/state:sa/federal_electorate:barker,South Australia - Federal House of Representatives Barker electorate,,
+ocd-division/country:au/state:sa/federal_electorate:boothby,South Australia - Federal House of Representatives Boothby electorate,,
+ocd-division/country:au/state:sa/federal_electorate:grey,South Australia - Federal House of Representatives Grey electorate,,
+ocd-division/country:au/state:sa/federal_electorate:hindmarsh,South Australia - Federal House of Representatives Hindmarsh electorate,,
+ocd-division/country:au/state:sa/federal_electorate:kingston,South Australia - Federal House of Representatives Kingston electorate,,
+ocd-division/country:au/state:sa/federal_electorate:makin,South Australia - Federal House of Representatives Makin electorate,,
+ocd-division/country:au/state:sa/federal_electorate:mayo,South Australia - Federal House of Representatives Mayo electorate,,
+ocd-division/country:au/state:sa/federal_electorate:port_adelaide,South Australia - Federal House of Representatives Port Adelaide electorate,,2019

```

Comment:
Are there more specific dates we can use than just the year?

Reply:
Thanks for calling it out. When cross checking with https://www.aec.gov.au/Electorates/Redistributions/dates.htm, the dates don't line up with the WIkipedia list, so I modified it to match the dates that are available in the AEC website.

The exact dates are available on the site by selecting the state -> Media Releases > "Names and boundaries of federal electoral divisions in <state> decided". For example: https://www.aec.gov.au/media/2021/06-28.htm